src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,199 @@
+
+package Duke;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
While the imports are fairly organised, it would be even better if there were line breaks in between groups of imports such as a group for io, then nio, and so on. As well a line break in between imports and package statements